deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/QueryRequestParser.java

```diff
@@ -13,9 +13,6 @@
 public class QueryRequestParser
 {
     private static final String QUERY_TAG = "query";
-    @Expose(deserialize = false)
-    @SerializedName(QUERY_TAG)
-    private String query = null;
```

Comment:
This change seems wrong to me. This is a json field declaration that is used for serialization/deserialization. I don't think you can make this a local field

Reply:
This is not being referenced anywhere though!

Reply:
It is accessed via reflection when converting to and from json though, so this isn't safe to remove.

Reply:
I'd just suppress the warning on this instance